setup.py

```diff
@@ -17,6 +17,7 @@
     "boto3==1.9.88",
     "botocore==1.12.116",
     "click==6.7",
+    "clint==0.5.1",
```

Comment:
Why do we have to specify it in both requirements and then in setup.py? Can we please use whatever is there in requirements in setup file for building wheel?